Add :LeanPlainTermGoal in case someone wants inline-hover term goals.

Julian opened the pull request:
Demo:

[![asciicast](https://asciinema.org/a/7gZzqLn1JfoykRy23EAL5iIDw.svg)](https://asciinema.org/a/7gZzqLn1JfoykRy23EAL5iIDw)

gebner commented on lua/lean/lsp.lua:
```diff
@@ -33,4 +40,18 @@ function M.handlers.plain_goal_handler (_, method, result, _, _, config)
+  if vim.tbl_isempty(markdown_lines) then
+    return
+  end
+  return vim.lsp.util.open_floating_preview(markdown_lines, "leaninfo", config)
```
Why is the goal converted to markdown if the filetype is leaninfo?

Julian replied:
That's me bad-copypasting from the above. Will fix.

gebner reviewed (commented):
This looks great!  There should probably be an entry in the README for the new command though.

Julian commented:
The README needs some love (which Patrick pointed out as well) -- I had some local changes on a start there, but for now, just added two lines on the two current commands so they're at least mentioned (and fixed the unnecessary markdownizing).
